.github/workflows/pull-request.yml

```
@@ -10,6 +10,6 @@ jobs:
   call_workflow:
     name: Run PR Build Workflow
     if: ${{ github.repository_owner == 'ballerina-platform' }}
-    uses: ballerina-platform/ballerina-library/.github/workflows/pr-build-connector-template.yml@main
+    uses: ballerina-platform/ballerina-library/.github/workflows/pr-build-connector-template.yml@fix-connector-pr-build
```

```suggestion
    uses: ballerina-platform/ballerina-library/.github/workflows/pr-build-connector-template.yml@main
```

This change was done for testing the workflow changes, please change it back once this PR is merged: https://github.com/ballerina-platform/ballerina-library/pull/6539